### ISSUE TITLE
Provisioning from latest checkout fails

### ISSUE THREAD

**Issue by sudar** [Automattic/vip-quickstart#379]
Provisioning from latest checkout fails with the following error

```
==> default: notice: /Stage[main]//Wp::Plugin[log-viewer]/Wp::Command[plugin activate log-viewer]/Exec[/srv/www/wp wp plugin activate log-viewer --network]/returns: executed successfully
==> default: notice: /Stage[main]//Package[pear.php.net/PHP_CodeSniffer]/ensure: created
==> default: notice: /Stage[main]//Vcsrepo[/usr/share/php/PHP/CodeSniffer/Standards/WordPress]/ensure: Creating repository from present
==> default: notice: /Stage[main]//Vcsrepo[/usr/share/php/PHP/CodeSniffer/Standards/WordPress]/ensure: created
==> default: notice: Finished catalog run in 1159.29 seconds
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

Is there anyway to fix this?

**Comment by joshbetz** [comment]
Can you paste the full log in a gist and link it here?

**Comment by sudar** [comment]
Here is the full output when I do `vagrant up` on a fresh (recursive) checkout of vip-quickstart

https://gist.github.com/sudar/d362472ef9b1f6b6ffef

I am doing this on Mac book pro running yosemite.

**Comment by joshbetz** [comment]
I don't see any errors. What problem are you having?

**Comment by sudar** [comment]
You see the error at the end of the gist file https://gist.github.com/sudar/d362472ef9b1f6b6ffef#file-gistfile1-txt-L468 (which I have already pasted at the first comment)

When I do `vagrant up` on a fresh checkout, the provisioning script stops after the above error message.

**Comment by joshbetz** [comment]
Right, it says that because PHPUnit failed to install, but everything else should work.

**Comment by sudar** [comment]
Since I got the "non-zero exit status" message, I assumed something else went wrong. Didn't realize that the non-zero exist status was only because of the PHPunit error above and the provisioning script has in fact completed.

BTW is there any way to fix the PHPunit error? I also created #378 about it.

Feel free to close it.
